src/video/windows/SDL_windowswindow.c

```diff
@@ -255,13 +261,18 @@ void WIN_SetWindowPositionInternal(SDL_Window *window, UINT flags)
     WIN_AdjustWindowRect(window, &x, &y, &w, &h, SDL_TRUE);
 
     data->expected_resize = SDL_TRUE;
-    SetWindowPos(hwnd, top, x, y, w, h, flags);
+    if (SetWindowPos(hwnd, top, x, y, w, h, flags) == 0) {
```

Comment:
This returns a BOOL value and should be tested as a boolean.

Reply:
I followed other examples in this file :)
I'll modify this one, and fix the others in a follow-up commit.

Reply:
Oh, if there are other functions doing it differently, that's fine, you can leave it as-is.